CHANGES.rst

```diff
@@ -9,6 +9,8 @@ ramp_fitting
 
 - Updating the one good group ramp suppression handler works. [#92]
 
+- Adding special case handler for GLS to handle one group ramps. [#97]
```

Comment:
I believe this should go into the "unreleased" section of the file. 0.7.3 is already out.